### PR TITLE
ADD module misc_settings

### PR DESCRIPTION
**Pull request by bealdav**
screen

### CODE REVIEW

**Comment by rvalyi** [comment]
nothing against the specific_settings module name. But I also suggest these names:

- custom_settings
- extra_settings
- misc_settings

**Comment by bealdav** [comment]
misc_settings seems good to me.

Let 's others give their idea

File: misc_settings/readme/CONFIG.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+You can override like here
+
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="model">res.config.settings</field>
+        <field name="priority" eval="200"/>
+        <field name="inherit_id" ref="misc_settings.res_config_settings_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[hasclass('app_settings_block')]" position="inside">
+                <div>
+                    Complet here
```
Comment (rvalyi) [inline misc_settings/readme/CONFIG.rst]: typo: Complet -> Complete

**Review by rvalyi — DISMISSED** [review]

File: misc_settings/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Misc settings to be an anchor for your custom settings
+
+Use case : you want your users find in one a unique dedicated page the settings they need rule their activity over the time.
```
Comment (kevinkhao) [inline misc_settings/readme/DESCRIPTION.rst:3]: LGTM but clarity suggestion

Use case: allow your users to manage any settings that you want to add

**Review by kevinkhao — APPROVED** [review]

File: misc_settings/views/res_config_settings.xml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="model">res.config.settings</field>
+        <field name="priority" eval="200" />
+        <field name="inherit_id" ref="base.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[hasclass('settings')]" position="inside">
+                <div
+                    class="app_settings_block"
+                    data-string="Misc"
+                    string="Misc"
```
Comment (carmenbianca) [inline misc_settings/views/res_config_settings.xml]: This should be 'Miscellaneous', the full English word.

**Review by carmenbianca — CHANGES_REQUESTED** [review]
LGTM, one comment

File: misc_settings/views/res_config_settings.xml (hunk 3 above)
Comment (bealdav) [reply misc_settings/views/res_config_settings.xml]: Fixed @carmenbianca Thanks for the review

File: misc_settings/i18n/fr.po [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"X-Generator: Poedit 3.0.1\n"
+
+#. module: misc_settings
+#: model_terms:ir.ui.view,arch_db:misc_settings.res_config_settings_view_form
+msgid "Misc"
+msgstr "Divers"
```
Comment (carmenbianca) [inline misc_settings/i18n/fr.po]: The French translation is now out-of-date.

File: misc_settings/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Misc Settings",
```
Comment (carmenbianca) [inline misc_settings/__manifest__.py]: Maybe also change this to 'Miscellaneous'. The technical module name `misc_settings` is good, though.

File: misc_settings/readme/DESCRIPTION.rst [hunk 6]
```diff
@@ -0,0 +1,6 @@
+Misc settings to be an anchor for your custom settings
```
Comment (carmenbianca) [inline misc_settings/readme/DESCRIPTION.rst]: Idem.

**Review by carmenbianca — CHANGES_REQUESTED** [review]
Sorry for the tiny details :sweat_smile:

**Review by carmenbianca — APPROVED** [review]
LGTM, thanks @bealdav !

**Pull request merged**